examples/opensora_hpcai/opensora/models/layers/blocks.py

```diff
@@ -158,15 +158,15 @@ def construct(self, x, cond, mask=None):
 
         # kv: (B N_k C*2) -> (B N_k 2 C) -> (B N_k 2 num_head head_dim).
         kv = ops.reshape(kv, (B, N_k, 2, self.num_heads, self.head_dim))
-        k, v = ops.split(kv, 1, axis=2)
+        k, v = mint.split(kv, 1, 2)
         # (b n h d)
         k = ops.squeeze(k, axis=2)
         v = ops.squeeze(v, axis=2)
 
         # 2+: mask adaptation for multi-head attention
         if mask is not None:
             # flip mask, since ms FA treats 1 as discard, 0 as retain.
-            mask = 1 - mask
+            mask = ops.logical_not(mask.to(ms.bool_)).to(ms.uint8)
```

Comment:
why change this?

Reply:
Refer to https://github.com/mindspore-lab/mindone/pull/585#discussion_r1671442440

Reply:
I mean 1 - mask (uint8) = 0 (uint8), exactly same as the new version, not related to the FA mask

Reply:
I understand what you mean, but I am not the author of the PR. You'd better ask @yuedongli1 for the reason behind this change in his PR #585.

Reply:
lol, fine